packages/windows/src/RPA/Windows/keywords/locators.py

```diff
@@ -241,6 +242,17 @@ def _get_element_with_locator_part(
         new_element.robocorp_click_offset = offset
         return new_element
 
+    def _load_by_alias(self, criteria: str) -> str:
+        try:
+            locator = LocatorsDatabase.load_by_name(criteria, self._locators_path)
+            if isinstance(locator, WindowsLocator):
+                return locator.value
```

Comment:
So if the obtained locator through criteria isn't found (or anything that makes it to not be an instance of `WindowsLocator `), then the criteria itself is returned (as the original passed locator), right? (wanted to make sure I understand it)

Reply:
That's right. I know the naming is a bit confusing there as the str value of criteria can be a raw locator string as well. There is no way of knowing though if it's a criteria (key in locators.json) or a raw locator value (for example: name:'Number pad' > name:'Nine').